README.md

```diff
@@ -23,14 +23,21 @@ jobs:
         id: ci
         uses: ignition-tooling/ubuntu-bionic-ci-action@v1
         with:
-          apt-dependencies: ''
           codecov-token: ${{ secrets.CODECOV_TOKEN }}
           cmake-args: '-DBUILD_TESTING=1'
 ```
 
 ### Dependencies
 
-Be sure to put all apt-installable dependencies into `apt-dependencies`.
+#### APT dependencies
+
+Be sure to declare all apt-installable dependencies in a flat list into:
```

Comment:
I think rather than "flat list", saying "one apt package per line" would be a bit clearer.